docsrc/tutorials/installation.rst

```diff
@@ -238,6 +238,34 @@ To build using the pre-CXX11 ABI use the ``pre_cxx11_abi`` config
 
     bazel build //:libtorchtrt --config pre_cxx11_abi -c [dbg/opt]
 
+**Building with CMake**
+-----------------------
+
+It is possible to build the API libraries (in cpp/) and the torchtrtc executable using CMake instead of Bazel.
+Begin by installing CMake.
+
+    * Latest releases of CMake and instructions on how to install are available for different platforms 
+      [on their website](https://cmake.org/download/).
+
+A few useful CMake options include:
+
+    * CMake finders for TensorRT and cuDNN are provided in `cmake/Modules`. In order for CMake to use them, pass
+      `-DCMAKE_MODULE_PATH=cmake/Modules` when configuring the project with CMake.
+    * Libtorch provides its own CMake finder. In case CMake doesn't find it, pass the path to your install of 
+      libtorch with `-DTorch_DIR=<path to libtorch>/share/cmake/Torch`
```

Comment:
Does this build system create something similar for TorchTRT? It would be nice if we could include this in the builds like we do for bazel.

Reply:
Yes, it does. That's done [here](https://github.com/gcuendet/Torch-TensorRT/blob/700831fe15a09b6a3c4048bb6a79280667e03300/CMakeLists.txt#L31). It basically generates a `torchtrtConfig.cmake` that allows a consumer to do 
```
find_package(torchtrt)
``` 
in his CMakeLists.txt and then use the Cmake targets directly.

Reply:
This behavior is now demonstrated in the examples CMakeLists.txt. I was added in [this commit](https://github.com/pytorch/TensorRT/pull/1058/commits/61bfec8e75a1312c8fb5481245d453296e446386).